### ISSUE TITLE
🚀 Feature: Doubts about updating emojilib to version 4

### ISSUE THREAD

**Issue by lucasvieceli** [omnidan/node-emoji#206]
### Bug Report Checklist

- [X] I am using the latest version of the package
- [X] I have [searched for related issues](https://github.com/omnidan/node-emoji/issues?q=is%3Aissue) and found none that matched my issue.

### Overview

What problem occurs when updating package emojilib lib to version 4?

**Comment by timbo-tj** [comment]
Yeah, I am finding a lot of common emojis are not picked up correctly by this lib, ie `emoji.which("🫡")` returns undefined.

**Comment by Chailotl** [comment]
This seems rather important, as even emojis from 2019 aren't being picked up i.e. `:blue_square:` 🟦
